main.go

```diff
@@ -85,6 +86,24 @@ var (
 		},
 	}
 
+	catwalkCmd = &cobra.Command{
+		Use:   "catwalk",
+		Short: "Opens the specified app in Catwalk",
+		Long: `Opens the app in Catwalk. Example: corectl catwalk --app my-app.qvf
```

Comment:
There is an `Example` section in a cobra command. Think we should start using it at least for new commands.